app/config/FrontendAppConfig.scala

```diff
@@ -68,6 +68,8 @@ class FrontendAppConfig @Inject() (val configuration: Configuration) {
 
   lazy val trustsStoreUrl: String = configuration.get[Service]("microservice.services.trusts-store").baseUrl + "/trusts-store"
 
+  lazy val agentInvitationsUrl: String = configuration.get[Service]("microservice.services.agent-invitations-frontend").baseUrl + "/invitations/agents/client-type"
```

Comment:
I suspect this won't work when we attempt to override in app-config?
This will need to be similar to Trust IV redirect config and contact-frontend. A single string.